README.md

```diff
@@ -0,0 +1,34 @@
+![qcon][qcon-logo]
+
+![linkerd][l5d-logo]
+
+:balloon: Welcome to the QCon Linkerd Workshop! :wave:
+
+## Intro
+
+This workshop contains three parts, each focusing on a Linkerd feature:
+* [circuit-breaking](circuit-breaking)
+* [load-balancing](load-balancing)
+* [tls](tls)
+
+## Initial Setup
+
+You will need the following on your system:
+1) git
+2) Docker
+
+### Clone repo
+
+```bash
+git clone https://github.com/adleong/qcon-linkerd-workshop.git
+```
+
+## Further reading
+
+Linkerd provides many more features. For further reading, head over to
+[linkerd.io](https://linkerd.io). For more examples, check out the
```

Comment:
I would maybe even link straight to linkerd.io/features so they can jump right in to reading!